plugins/image-source/obs-slideshow.c

```diff
@@ -933,6 +940,9 @@ static obs_properties_t *ss_properties(void *data)
 					 file_filter, path.array);
 	dstr_free(&path);
 
+	obs_properties_add_int(ppts, S_MAX_MEMORY_USAGE, T_MAX_MEMORY_USAGE, 0,
```

Comment:
Rather than (megabytes) in the displayed string, use `obs_property_int_set_suffix()` to display it in the input (ideally as `" MB"`).

Reply:
Oh, cool, added the implementation :)